src/components/volume-viewer/VolumeRenderer.js

```diff
@@ -75,7 +76,7 @@ export default connect(mapStateToProps)(
               cursor-listener
             />
 
-            <VolumeConsumer>
+            <SelectorConsumer>
               {({ state }) => (
```

Comment:
As we think about pulling out the renderer as a standalone package/component, we'll probably want to pass in the consumer values as props a level up instead of inside the component, (or more the component a level down). That doesn't need to be addressed at this point though.